scripts/imageIDI.py

```diff
@@ -265,6 +289,8 @@ def main(args):
                         help='disable source and grid labels')
     parser.add_argument('-g', '--no-grid', action='store_true', 
                         help='disable the coordinate grid')
+    parser.add_argument('-v', '--lwasv', action='store_true',
+                        help='set phase center to report correctly for LWA-SV')
```

Comment:
Shouldn't this already be handled by the metadata in the FITS-IDI file?

Reply:
Done